### PR TITLE
Clarifications in the suspension/removal text

### PR DESCRIPTION
**Pull request by frivoal**
See #312

***
<a href="https://pr-preview.s3.amazonaws.com/frivoal/w3process/pull/528.html" title="Last updated on May 12, 2021, 7:15 AM UTC (97be7a7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/w3process/528/5b02c16...frivoal:97be7a7.html" title="Last updated on May 12, 2021, 7:15 AM UTC (97be7a7)">Diff</a>

### CODE REVIEW

**Review by nigelmegitt — APPROVED** [review]

File: index.bs [hunk 1]
```diff
@@ -1020,9 +1020,12 @@ Expectations and Discipline</h4>
 	including suspending or removing for cause
 	a participant in any group (including the [=AB=] and [=TAG=])
 	if serious and/or repeated violations,
-	such as failure to meet the requirements of this process
+	such as failure to meet the requirements on individual behavior of
+	(a) this process
 	(where repeated attempts to address these violations have not resolved the situation)
```
Comment (dwsinger) [inline index.bs:1025]: I don't think we should repeat "repeated" here, it's in the beginning. and if the violation is sufficiently serious, as the sentence starts, once may be enough.

Comment (fantasai) [reply index.bs:1025]: Good point. :) I think we needed the parenthetical in the old wording, but not since we restructured this section.

Comment (frivoal) [reply index.bs]: Agreed. removed the parenthetical.

File: index.bs [hunk 2]
```diff
@@ -1020,9 +1020,12 @@ Expectations and Discipline</h4>
...
-	or violation of the membership agreement or applicable laws,
+	and in particular the CEPC, or
```
[the remark this answers is not in the document]
Comment (frivoal) [reply index.bs:1025]: Removed the parenthetical, so this problem solved itself.

**Pull request merged**